Source/WebKit/Shared/API/Cocoa/_WKRemoteObjectRegistry.mm

```diff
@@ -295,8 +297,9 @@ - (void)_invokeMethod:(const WebKit::RemoteObjectInvocation&)remoteObjectInvocat
             auto encoder = adoptNS([[WKRemoteObjectEncoder alloc] init]);
             [encoder encodeObject:invocation forKey:invocationKey];
 
-            if (remoteObjectRegistry->_remoteObjectRegistry)
-                remoteObjectRegistry->_remoteObjectRegistry->sendReplyBlock(replyID, WebKit::UserData([encoder rootObjectDictionary]));
+            RefPtr protectedRemoteObjectRegistry = remoteObjectRegistry->_remoteObjectRegistry;
```

Comment:
ditto.

Reply:
I removed `protected...` from the other locations but had to keep it here since `remoteObjectRegistry` is already being used in this scope.